.github/sandbox_workflows.md

```
@@ -47,57 +45,19 @@ New Feature: Before starting the pipeline execution, the workflow checks if secr
-  The variable specifies the maximum allowed age of a token in seconds, after which the token is considered expired and requires rotation.
-  Optional:
-    If not provided, the default value of 601200 seconds (approximately 7 days) will be used.
-
-  Name: TOKEN_MAX_AGE_SECONDS
-  Value: 601200
+- Navigate to **Settings > Secrets and variables > Actions > Variables** in your GitHub organization.
+- Add the following variables:
+  - `TEST_AWS_ACCOUNT_ID`: The ID of the AWS account for token rotation in the test environment.
+  - `PROD_AWS_ACCOUNT_ID`: The ID of the AWS account for token rotation in the prod environment.
+  - `AWS_REGION`: The region of the AWS account.
+  - [`GITHUB_TOKEN`](https://github.com/VilnaCRM-Org/website-infrastructure/blob/main/.github/github-token-usage.md): The token for getting PR number.
```

markdownlint-cli2 (0.17.2)

Line length
Expected: 100; Actual: 150

(MD013, line-length)